rl_zoo3/train.py

```diff
@@ -153,6 +153,13 @@ def train() -> None:
         default=False,
         help="if toggled, display a progress bar using tqdm and rich",
     )
+    parser.add_argument(
```

Comment:
Missing: a section in the readme, I will try to create a real doc soon (the readme is way too long now).

Reply:
doc is there